src/Controls/src/Core/HandlerImpl/RoundRectangle.Impl.cs

```diff
@@ -51,10 +52,14 @@ public override PathF GetPath()
 			float w = (float)(width - StrokeThickness);
 			float h = (float)(height - StrokeThickness);
 
-			float topLeftCornerRadius = (float)CornerRadius.TopLeft;
-			float topRightCornerRadius = (float)CornerRadius.TopRight;
-			float bottomLeftCornerRadius = (float)CornerRadius.BottomLeft;
-			float bottomRightCornerRadius = (float)CornerRadius.BottomRight;
+			float density = 1f;
+#if ANDROID
+			density = (float)DeviceDisplay.MainDisplayInfo.Density;
```

Comment:
Reviewing why we need this changes and if have more sense to apply it at other level. We have not used density before.

Reply:
Maybe related to https://github.com/dotnet/maui/commit/982d809eb9567f75ea810a3d7aedb6aac8f8284c#diff-9962b2ab4d4a0eb4922307e668bfe7d868c4fac7919a9463a3cba1859a6de86fL417-L423